pd/src/genesis.rs

```diff
@@ -47,15 +47,21 @@ impl Allocation {
     }
 }
 
+#[derive(Deserialize, Serialize, Debug, Clone)]
+pub struct ValidatorPower {
+    pub validator: Validator,
+    pub power: tendermint::vote::Power,
+}
+
```

Comment:
Now that the validator power isn't part of the `Validator` struct (since it's not an inherent attribute of the validator, but a contextual one), I needed a way to specify the validator power in the genesis file, but there's otherwise no principled reason for this to exist.

It might be more useful if the `stake` crate had a `ValidatorState` type that could record the validator's voting power, current state (is it in the consensus set? was it slashed? etc), which could then be used by the genesis data.